src/pages/profiles/Profile.js

```diff
@@ -0,0 +1,207 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { toast } from 'react-toastify';
+import SideBar from '../../components/SideBar.js';
+import { ButtonA as Button } from '../../components/Button.js';
+import { saveInfo, editInfo } from '../../redux/reducers/profileReducer.js';
+import profileImg from '../../images/profileImg.jpg';
+import pen from '../../assets/pen-solid.svg';
+import Input from '../../components/Input.js';
+
+const Profile = ({ firstName,lastName, company, mobileNumber,
+  address, dob, id, position, email }) => {
+    
+  const [src, setSrc] = useState(profileImg);
+  const dispatch = useDispatch();
+  const {
+    disabled,
+    styles,
+    hideSave,
+    upload,
+    editIcon,
+    editPicture,
+    hideEdit
+  } = useSelector((state) => state?.profileInfo);
+
+  const onChange = (e) => {
+    const reader = new FileReader();
+    const file = e.target.files[0];
+    reader.onloadend = () => {
+      setSrc(reader.result);
+    };
+    reader.readAsDataURL(file);
+  };
+
+  return (
+    <div className="flex h-full font-sans antialiased bg-grey-lightest w-full">
+      <section>
+        <SideBar />
+      </section>
+      <section className="mx-auto items-center w-full h-full p-8 top-8 right-2 ">
+        <h1 className="font-black text-2xl">Profile</h1>
+        <div className="bg-white rounded p-4 w-full shadow-md min-h-[70vh] mt-6 pb-10">
+          <div>
+            <div className="flex flex-col tex-center md:flex-row lg:flex-row xl:flex-row items-center justify-center bg-gray-100 mb-10 h-80 md:h-60 lg:h-60 xl:h-60 pb-10">
+              <span className="w-20 h-20 my-8 md:w-24 lg:w-24 xl:w-24 md:h-24 lg:h-24 xl:h-24 mr-6">
+                <div className="relative">
+                  <label htmlFor={upload}>
+                    <img
+                      src={src}
+                      alt="profile-img"
+                      id="profile-img"
+                      className={`${editPicture} rounded-full object-cover w-20 h-20 md:w-24 lg:w-24 xl:w-24 md:h-24 lg:h-24 xl:h-24`}
+                    />
+                    <div className="edit">
+                      <img
+                        src={pen}
+                        alt=""
+                        className={`${editIcon} absolute left-7 md:left-10 lg:left-10 xl:left-10 top-7 md:top-8 lg:top-8 xl:top-8 w-6 cursor-pointer`}
```

Comment:
This icon hide the entire profile picture as it is located in the middle.

Change the icon to occupy the better part of the bottom of the picture. Maybe look into options to show the icon on hover only and see how it would play out.